deepquest/utils/prepare_data.py

```diff
@@ -9,6 +9,74 @@
 # We patch the Dataset class here.
 # This will not be necessary when multimodal_keras_wrapper is fixed.
```

Comment:
This comment is supposed to be attached to, and precede, the `dataset_preprocessreal_fixed` function. But you slapped a gigantic new function in between this comment and the function that follows. I suppose that's my fault for putting blank lines between them.

Reply:
Quite right! Have actually just removed `dataset_preprocessreal_fixed` in a6c9376 since this is now fixed in multimodal_keras_wrapper.